### PR TITLE
fix(ui): Handle displaying TopCounter if user is not allowed

### PR DESCRIPTION
**Pull request by Jordan91**
## Description

Handle displaying TopCounter if user is not allowed

**Fixes** # (issue)

Remove loading of the Skeleton and Bypass Error in Snackbar with http error code 401

## Type of change

- [x] Patch fixing an issue (non-breaking change)

Header without skeleton

## Target serie

- [x] 21.04.x
- [x] 21.10.x
- [x] 22.04.x (master)

<h2> How this pull request can be tested ? </h2>

Log in the UI with user without action access to Display top counter (Poller / Host / Service / Bam) , and you must haven't any top counter.

### CODE REVIEW

File: www/front_src/src/Header/PollerMenu/index.tsx [hunk 1]
```diff
@@ -134,6 +138,10 @@ const PollerMenu = (): JSX.Element => {
     setToggled(!toggled);
   };
 
+  if (!isAllowed) {
+    return <div />;
```
Comment (Jordan91) [inline www/front_src/src/Header/PollerMenu/index.tsx:142]: null

Comment (Jordan91) [inline www/front_src/src/Header/PollerMenu/index.tsx:142]: `null`

File: www/front_src/src/Header/RessourceStatusCounter/index.tsx [hunk 2]
```diff
@@ -51,6 +52,8 @@ const RessourceStatusCounter = <
       })
       .catch((error) => {
         if (error.response && error.response.status === 401) {
+          setIsAllowed(false);
```
Comment (bdauria) [inline www/front_src/src/Header/RessourceStatusCounter/index.tsx:55]: Same remark as above

File: www/front_src/src/Header/PollerMenu/index.tsx [hunk 3]
```diff
@@ -125,6 +127,8 @@ const PollerMenu = (): JSX.Element => {
       })
       .catch((error) => {
         if (error.response && error.response.status === 401) {
+          setIsAllowed(false);
+        } else {
           setIssues(null);
         }
```
[the remark this answers is not in the document]
Comment (Jordan91) [reply www/front_src/src/Header/PollerMenu/index.tsx:133]: Thank's , changes made and i check if it's still ok to push again

File: www/front_src/src/Header/RessourceStatusCounter/index.tsx (hunk 2 above)
Comment (Jordan91) [reply www/front_src/src/Header/RessourceStatusCounter/index.tsx:55]: Thank's , changes made and i check if it's still ok to push again

**Review by Thebarda — DISMISSED** [review]

**Review by bdauria — DISMISSED** [review]

**Review by bdauria — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]

**Pull request merged**